howfairis/repo.py

```diff
@@ -116,5 +116,5 @@ def _get_default_branch(self):
         try:
             response.raise_for_status()
         except requests.HTTPError as e:
-            raise GetDefaultBranchException("Something went wrong asking the repo for its default branch.") from e
+            raise GetDefaultBranchException("Something went wrong asking the repo for its default branch: " + str(e)) from e
```

Comment:
Why did you add the `str(e)` here?

Reply:
Because the error message was not clear enough. During testing this error occurred and only after including str(e) here, I could find out what was wrong. I hope this addition will save other testers some time.

Reply:
Ok, can you explain what `from e` means in the snippet above?